lib/services/ios-project-service.ts

```diff
@@ -910,28 +970,44 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	public checkForChanges(changesInfo: IProjectChangesInfo, options: IProjectChangesOptions, projectData: IProjectData): void {
-		const provision = options.provision;
-		if (provision !== undefined) {
+	public async checkForChanges(changesInfo: IProjectChangesInfo, {provision, teamId}: IProjectChangesOptions, projectData: IProjectData): Promise<void> {
+		if (provision !== undefined || teamId !== undefined) {
 			// Check if the native project's signing is set to the provided provision...
 			const pbxprojPath = this.getPbxProjPath(projectData);
 
 			if (this.$fs.exists(pbxprojPath)) {
 				const xcode = this.$pbxprojDomXcode.Xcode.open(pbxprojPath);
 				const signing = xcode.getSigning(projectData.projectName);
-				if (signing && signing.style === "Manual") {
-					for (let name in signing.configurations) {
-						let config = signing.configurations[name];
-						if (config.uuid !== provision && config.name !== provision) {
-							changesInfo.signingChanged = true;
-							break;
+
+				if (provision !== undefined) {
+					if (signing && signing.style === "Manual") {
+						for (let name in signing.configurations) {
```

Comment:
We can, but enabling it leads to 1400 compile errors, so I advise against this in this PR